api-tests/features/p2-m2b.feature

```diff
@@ -0,0 +1,34 @@
+@Build
+Feature: M2B No Photo Id Journey
+
+  Scenario: M2B Journey
+    Given I start a new 'medium-confidence' journey
+    Then I get a 'page-ipv-identity-document-start' page response
+    When I submit an 'end' event
+    Then I get a 'page-ipv-identity-postoffice-start' page response
+    When I submit an 'end' event with feature set 'm2bBetaExperianKbv'
+    Then I get a 'prove-identity-no-photo-id' page response
+    When I submit an 'next' event
+    Then I get a 'claimedIdentity' CRI response
+    When I submit 'kenneth-current' details to the CRI stub which had request attributes
```

Comment:
Mike TODO: When I submit 'kenneth-current' details to the CRI stub and see requested context 'bank_account'